### PR TITLE
Document GitHub Status check integration.

### PR DESCRIPTION
**Pull request by adamzimmermann**
## Description
Documents how to integrate this tool with GitHub.

## Motivation / Context
I struggled to remember how to set this up this morning, so I figured it was worth helping the next person.

## Testing Instructions / How This Has Been Tested
N/A

## Documentation
You are looking at it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,18 +16,29 @@ downstream repos.
 
 ## Configuration
 
-1. Create a `robo.yml` file in the root of your codebase. `robo.drupal.example.yml`
+- Create a `robo.yml` file in the root of your codebase. `robo.drupal.example.yml`
```
Comment (adamzimmermann) [inline README.md:19]: The order didn't seem like it mattered for these, so I changed them. I can be talked into reverting that though.

**Review by adamzimmermann — COMMENTED** [review]
@goranmiric I would love your take on this as someone who didn't help build this.

File: README.md [hunk 2]
```diff
@@ -16,18 +16,29 @@ downstream repos.
...
     "robo": "robo --ansi"
-}
-```
+  }
+  ```
+- If you are overriding any commands, add the following to your repo's `composer.json` file.
+  ```json
+  "autoload": {
+  "psr-4": {
+      "BPRobo\\": "robo/src"
```
Comment (markdorison) [inline README.md]: This and the reference to BPRobo below should be removed. This is going to depend on whatever namespace the consumer is setting.

File: README.md [hunk 3]
```diff
@@ -16,18 +16,29 @@ downstream repos.
...
+    }
+  }
+  ```
+  Then place the commands in `robo/src/Robo/Plugin/Commands` with the
+  `BPRobo\Robo\Plugin\Commands` namespace.
+- If you are using any of the commands that use the [`GitHubStatusTrait`](src/Robo/Plugin/Traits/GitHubStatusTrait.php), a personal access token must be configured in GitHub.
```
Comment (markdorison) [inline README.md]:
```suggestion
- If you are using any of the commands that use the [`GitHubStatusTrait`](src/Robo/Plugin/Traits/GitHubStatusTrait.php) to set status checks on your pull requests, a personal access token must be created in GitHub.
```

File: README.md [hunk 4]
```diff
@@ -16,18 +16,29 @@ downstream repos.
...
+   1. Create a [personal access token](https://github.com/settings/tokens/new) in GitHub.
+   1. Create an environment variable named `GITHUB_ACCESS_TOKEN` with the token value.
```
Comment (markdorison) [inline README.md]:
```suggestion
   1. Create an environment variable named `GITHUB_ACCESS_TOKEN` with the token value in the environment where the checks will run.
```

**Review by markdorison — CHANGES_REQUESTED** [review]
This is great! I have requested a few changes.

File: README.md (hunk 2 above)
Comment (adamzimmermann) [reply README.md]: Ah, yes, I missed that initial in there!

Should it be `ExampleRobo` or something generic then?

File: README.md [hunk 5]
```diff
@@ -16,18 +16,29 @@ downstream repos.
...
-"scripts": {
+- Add the following to your repo's `composer.json` "scripts" section so that you can call robo easily with `composer robo`:
+  ```json
+  "scripts": {
     "robo": "robo --ansi"
-}
-```
+  }
+  ```
+- If you are overriding any commands, add the following to your repo's `composer.json` file.
```
Comment (markdorison) [inline README.md]:
```suggestion
- If you are overriding any commands, add the following to your repo's `composer.json` file. Be sure to specify your own namespace prefix in place of `YOURPREFIX`.
```

File: README.md (hunk 2 above)
Comment (markdorison) [reply README.md]:
```suggestion
      "YOURPREFIXRobo\\": "robo/src"
```

File: README.md [hunk 6]
```diff
@@ -16,18 +16,29 @@ downstream repos.
...
+  ```json
+  "autoload": {
+  "psr-4": {
+      "BPRobo\\": "robo/src"
+    }
+  }
+  ```
+  Then place the commands in `robo/src/Robo/Plugin/Commands` with the
+  `BPRobo\Robo\Plugin\Commands` namespace.
```
Comment (markdorison) [inline README.md]:
```suggestion
  `YOURPREFIXRobo\Robo\Plugin\Commands` namespace.
```

**Review by markdorison — CHANGES_REQUESTED** [review]
Updated suggestions!

**Review by markdorison — APPROVED** [review]

**Pull request merged**